scopes/component/component/ui/use-component-query.ts

```diff
@@ -110,11 +118,15 @@ const SUB_COMPONENT_REMOVED = gql`
 `;
 
 /** provides data to component ui page, making sure both variables and return value are safely typed and memoized */
-export function useComponentQuery(componentId: string, host: string) {
+export function useComponentQuery(
+  componentId: string,
+  host: string,
+  filters?: { log?: { logType?: string; logOffset?: number; logLimit?: number; logHead?: string } }
```

Comment:
this is a little redundant.

can you remove the "logXX" from the log filter properties?

Reply:
I did it to make spreading them convenient in the variables section
otherwise will have to map them individually
logHead: filters?.log?.logHead , ..etc